lego/apps/surveys/tasks.py

```diff
@@ -0,0 +1,30 @@
+from django.db import transaction
+from django.utils import timezone
+from structlog import get_logger
+
+from lego import celery_app
+from lego.apps.events.constants import PRESENT
+from lego.apps.stats.utils import track
+from lego.apps.surveys.models import Survey
+from lego.apps.surveys.notifications import SurveyNotification
+from lego.utils.tasks import AbakusTask
+
+log = get_logger()
+
+
+@celery_app.task(serializer='json', bind=True, base=AbakusTask)
+def send_survey_mail(self, logger_context=None):
+    self.setup_logger(logger_context)
+
+    surveys = Survey.objects.filter(active_from__lte=timezone.now(), sent=False)
+    print('surveys in send_mail', surveys.all())
+    for survey in surveys.all():
+        with transaction.atomic():
```

Comment:
I don't think you need this transaction wrapper. Since you're always sending out all the notifications *before* you set `sent = True` and save, it'll only save if you managed to send out all the surveys (so the transaction doesn't matter at all). 

What might happen regardless is that it can create notifications for half the users, then crash, and the next time it tries to do so it'll send out notifications for those users again - but the transaction doesn't help with that (you'd need one `sent` field per user).

(also remove the prints)